app/Console/Commands/RebuildImages.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Poniverse\Ponyfm\Console\Commands;
+
+use Illuminate\Console\Command;
+use Poniverse\Ponyfm\Models\Image;
+use Symfony\Component\HttpFoundation\File\File;
+
+class RebuildImages extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rebuild:images';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Resizes all images to fit the specifications in Models/Image';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $images = Image::all();
+        foreach ($images as $image) {
+            $this->info("Regenerating images for id:".$image->id. " (".$image->filename.")");
+            $image->clearExisting();
+
+            $originalFile = new File($image->getFile(Image::ORIGINAL));
```

Comment:
The `Image` class is a leaky abstraction here; this code would be cleaner if the CLI task could rebuild an `Image` object's on-disk image files without needing to be aware of the actual on-disk paths and file names - or without knowing anything about how the image files are stored, for that matter.

Try encapsulating lines 47-51 into an instance method on the `Image` class. The comment I left below about handling failures in the CLI call still applies here and as a general principle, by the way.